### PR TITLE
Add test of `dashboard` PR#7

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

Add test of `dashboard` PR#7
https://github.com/drucker/drucker-dashboard/pull/7

## This PR includes

- Check existence of dumped files

## What type of PR is it?

Feature

## What is the issue?

N/A

## How should this be tested?

Run unit test.

```
python -m unittest tests/drucker_dashboard/test_kubernetes.py
```

### CODE REVIEW

File: tests/drucker_dashboard/test_kubernetes.py [hunk 1]
```diff
@@ -68,6 +71,17 @@ def test_post(self):
         ).one_or_none()
         self.assertIsNotNone(kobj)
 
+        # Check Kubernetes dump files
+        aobj = Application.query.filter_by(
+            kubernetes_id=kobj.kubernetes_id).one_or_none()
+        if aobj is not None:
```
Comment (yoquankara) [inline tests/drucker_dashboard/test_kubernetes.py]: Can't we assert `aobj` is not `None` here?

Comment (keigohtr) [reply tests/drucker_dashboard/test_kubernetes.py]: No we can't.
I have noticed that this `aobj` is always `None`.
So we can remove this `# Check Kubernetes dump files` section.

Comment (yoquankara) [reply tests/drucker_dashboard/test_kubernetes.py]: Yeah, let's remove this section!

**Comment by keigohtr** [comment]
@yoquankara 
Thank you for the comment. I have removed an unnecessary section!

**Pull request merged**